input/otlp/logs.go

```diff
@@ -239,3 +245,28 @@ func (c *Consumer) convertLogRecord(
 
 	return event
 }
+
+// https://www.elastic.co/guide/en/ecs/current/ecs-data_stream.html
+func sanitizeDataStream(k string) string {
```

Comment:
```suggestion
func sanitizeDataStreamField(k string) string {
```

nit: sanitizeDataStream(k) makes me assume that k is the full data stream name e.g. `logs-foo-default`. To make it even more explanatory, it would be `func sanitizeDataStreamField(datasetOrNamespace string) string {`, but it doesn't look idiomatic. An alternative would be to add a comment to describe what this function does, and what it expects.